ncm-pbsserver/src/main/pan/components/pbsserver/schema.pan

```diff
@@ -139,6 +139,8 @@ type pbs_server_attlist = {
     'min_threads' ? long(0..)
 
     'legacy_vmem' ? boolean
+
+    'display_job_server_suffix' ? boolean
```

Comment:
Maybe I'm being stupid, but I can't see how this relates the change to the Perl module.

Reply:
Not, it's a different change. It shouldn't have been in this pull request.